config/config.go

```diff
@@ -500,10 +501,21 @@ type EnableSwitchWithPatches struct {
 	// Enabled defines if this option should be enabled.
 	Enabled bool `json:"enabled,omitempty"`
 
+	// Selector defines the labelSelector based filtering for syncing the resources to the host cluster.
+	Selector *SyncSelector `json:"selector,omitempty"`
```

Comment:
Adding this here would also apply to other resources that use EnableSwitchWithPatches, is that wanted?

Reply:
We can have a separate object specifically for this purpose. But I've added this in EnableSwitchWithPatches, keeping in mind that it would provide us an option to enable such selector based filtering for other resources as well. And the user can user let it be empty if they don't require any filtering.